<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-11T09:12:44 IST" name="Suite" finished-at="2025-04-11T09:13:59 IST" duration-ms="75686">
    <groups>
      <group name="Master">
        <method signature="TC002_LoginTest.VerifLogin()[pri:0, instance:testCases.TC002_LoginTest@64b73e7a]" name="VerifLogin" class="testCases.TC002_LoginTest"/>
      </group> <!-- Master -->
      <group name="Sanity">
        <method signature="TC002_LoginTest.VerifLogin()[pri:0, instance:testCases.TC002_LoginTest@64b73e7a]" name="VerifLogin" class="testCases.TC002_LoginTest"/>
      </group> <!-- Sanity -->
    </groups>
    <test started-at="2025-04-11T09:12:44 IST" name="chromeTest" finished-at="2025-04-11T09:13:59 IST" duration-ms="75686">
      <class name="testCases.TC002_LoginTest">
        <test-method is-config="true" signature="Setup(java.lang.String,java.lang.String)[pri:0, instance:testCases.TC002_LoginTest@64b73e7a]" started-at="2025-04-11T09:12:44 IST" name="Setup" finished-at="2025-04-11T09:13:48 IST" duration-ms="64003" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method signature="VerifLogin()[pri:0, instance:testCases.TC002_LoginTest@64b73e7a]" started-at="2025-04-11T09:13:49 IST" name="VerifLogin" finished-at="2025-04-11T09:13:58 IST" duration-ms="9802" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testCases.TC002_LoginTest@64b73e7a]" started-at="2025-04-11T09:13:59 IST" name="tearDown" finished-at="2025-04-11T09:13:59 IST" duration-ms="815" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testCases.TC002_LoginTest -->
    </test> <!-- chromeTest -->
  </suite> <!-- Suite -->
=======
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-01T10:57:11 IST" name="Default suite" finished-at="2025-04-01T10:57:11 IST" duration-ms="134">
    <groups>
    </groups>
    <test started-at="2025-04-01T10:57:11 IST" name="Default test" finished-at="2025-04-01T10:57:11 IST" duration-ms="134">
      <class name="Day46.OrangeHRM">
        <test-method signature="Openapp()[pri:1, instance:Day46.OrangeHRM@59662a0b]" started-at="2025-04-01T10:57:11 IST" name="Openapp" finished-at="2025-04-01T10:57:11 IST" duration-ms="27" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Openapp -->
        <test-method signature="Login()[pri:2, instance:Day46.OrangeHRM@59662a0b]" started-at="2025-04-01T10:57:11 IST" name="Login" finished-at="2025-04-01T10:57:11 IST" depends-on-methods="Day46.OrangeHRM.Openapp" duration-ms="5" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at Day46.OrangeHRM.Login(OrangeHRM.java:13)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
        <test-method signature="Search()[pri:3, instance:Day46.OrangeHRM@59662a0b]" started-at="2025-04-01T10:57:11 IST" name="Search" finished-at="2025-04-01T10:57:11 IST" depends-on-methods="Day46.OrangeHRM.Login" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method OrangeHRM.Search()[pri:3, instance:Day46.OrangeHRM@59662a0b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method OrangeHRM.Search()[pri:3, instance:Day46.OrangeHRM@59662a0b] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Search -->
      </class> <!-- Day46.OrangeHRM -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
>>>>>>> 32a351c65f697370896e02a232e42208e51e1396
</testng-results>
